foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigration.java

```diff
@@ -47,43 +48,45 @@ protected boolean shouldExecuteOnFile(File file) {
 
         try {
             if(file.getName().equals("aiops-security.properties")) {
-                BufferedReader aiopsReferencePythonPropertiesFileConfig = new BufferedReader((new FileReader(file)));
-
-                String lineOne;
-                while((lineOne = aiopsReferencePythonPropertiesFileConfig.readLine()) !=null && !shouldExecute) {
-                    for (String key : AIOPS_REFERENCE_PYTHON_PROPERTIES_FILE_PACKAGE_MAP.keySet()) {
-                        if (lineOne.contains(key)) {
-                            shouldExecute = true;
-                        }
-                    }
-                }
+                shouldExecute = true;
             } else {
                 BufferedReader aiopsReferencePythonDockerfileConfig = new BufferedReader((new FileReader(file)));
 
                 String lineTwo;
```

Comment:
Should just call this `line` since it updates with each iteration of the while loop.

Reply:
Renamed the variable.